src/main/java/net/sourceforge/tess4j/util/ImageIOHelper.java

```diff
@@ -58,11 +58,11 @@
 
     private static final Logger logger = LoggerFactory.getLogger(new LoggHelper().toString());
 
```

Comment:
@nguyenq the changes of this file are safe and makes sende for me, though crappy format.